api/api/src/main/scala/uk/ac/wellcome/platform/api/Router.scala

```diff
@@ -52,7 +52,7 @@ class Router(elasticClient: ElasticClient,
                 }
               },
               path("context.json") {
-                getFromFile(context.getPath)
```

Comment:
I put this in here because it's speculative: it might fix https://github.com/wellcometrust/platform/issues/4133. We have a test for whether `context.json` resolves, which doesn't fail, so I am guessing that the issue is related to how the resource is resolved in the running application... Not sure how to test it other than deploying it, and either way this is more succinct (even if it's still broken)